programs/editor/src-bellabello.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
Rough idea was to have a dedicated fuzzytesting collab app here. Without all the bling of having different documents session running at the same time, which the split screen editor demo has, at least as drafted in the code.

And that is also where there is a diff started against the original splitscreencode, all session selection/switching has been dumped already here, besides having the plugin hardcoded to load and having Bella and Bello identities for the users.

Next to do would be testing of convergence of documents for given stopping points etc. Doing this outside of a plugin seemed more complicated to implement for now, so I first wanted to solve this problem with no extra hurdles by having to work only out of plugins.

Perhaps one day this fuzzytesting collab demo can be turned into a plugin system for the normal splitscreen editor demo app again, that might be nice to have, yes. But top-priority for now for me is to have any fuzzytesting collab demo at all. So that's why there is this WIP.